.github/workflows/upload.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         run: |
           go run ./cmd/download-plugins -since ${{ inputs.since }} downloads
       - name: Upload To Release Bucket
-        run: gsutil rsync -r downloads gs://buf-plugins
+        run: gsutil -m rsync -r downloads gs://buf-plugins
```

Comment:
Saw this printed in logs:
```
==> NOTE: You are performing a sequence of gsutil operations that may
run significantly faster if you instead use gsutil -m rsync ... Please
see the -m section under "gsutil help options" for further information
about when gsutil -m can be advantageous.
```

Reply:
cool - looks like this is just parallelizing things? from `gsutil help options`:

```
  -m          Causes supported operations (acl ch, acl set, cp, mv, rm, rsync,
              and setmeta) to run in parallel. This can significantly improve
              performance if you are performing operations on a large number of
              files over a reasonably fast network connection.
```